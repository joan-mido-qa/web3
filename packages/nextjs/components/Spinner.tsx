export default function Spinner() {
	return (
		<div className="absolute right-1/2 bottom-1/2  transform translate-x-1/2 translate-y-1/2 ">
			<div className="border-t-transparent border-solid animate-spin  rounded-full border-blue-500 border-8 h-40 w-40"></div>
		</div>
	);
}
