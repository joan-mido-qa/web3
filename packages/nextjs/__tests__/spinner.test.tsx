import Spinner from "@/components/Spinner";
import { render } from "@testing-library/react";

describe("Spinner", () => {
	test("Should render Spinner component unchanged", async () => {
		const { container } = render(<Spinner />);
		expect(container).toMatchSnapshot();
	});
});
