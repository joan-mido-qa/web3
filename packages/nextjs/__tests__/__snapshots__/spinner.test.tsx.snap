// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Should render spinner component unchanged 1`] = `
<div>
  <div
    class="absolute right-1/2 bottom-1/2  transform translate-x-1/2 translate-y-1/2 "
  >
    <div
      class="border-t-transparent border-solid animate-spin  rounded-full border-blue-500 border-8 h-40 w-40"
    />
  </div>
</div>
`;
